_docs/protocols/2017-01-06-binary.md

```diff
@@ -215,7 +215,17 @@ ghci> hexEncode $ Data.HashMap.Strict.fromList [(1 :: Word8, 127 :: Word64), (2,
 newtype MessageName = MessageName BS.ByteString
 ~~~
 
-`MessageName` is encoded in same manner as `ByteString`.
+`MessageName` is encoded in same manner as `ByteString`. Note that encoded `ByteString` is itself encoded `UnsignedVarInt`, or concatination of two encoded
```

Comment:
Well, I understand what is supposed to be told to reader. But `UVarInt` is not encoded in the same way as `ByteString`. If length of `UvarInt` in bytes is added I think it worth mentioning here.